app/views/users/edit.html.slim

```diff
@@ -0,0 +1,16 @@
+h1.d-flex.justify-content-center
+  |  Edit Profile
+hr
+.d-flex.justify-content-center
+  = form_with model:@user do |f|
+    h2
```

Comment:
Trekns label priekš text fielda :D , bet h2 iztrūkst vienas lietas, kas nāk līdz ar html label lietojumu.

HTMLā, ja nospied uz label - tad kursors iefokusējas iekš input lauka, diemžēl ar h2 tā nav, tāpēc dubultā jāpārdomā vai izvēlētā struktūra ir atbilstoša HTML standartiem un vispārpieņemtajām praksēm.

Reply:
Bet pagaidām par to vari arī neiespringt

Reply:
Nomainiju uz label , tbh labak izsktas 😄